source/IntegrationEventListener/Energinet.DataHub.Aggregations.DatabaseMigration/Scripts/202203250901_Create_MarketRole_Table.sql

```diff
@@ -0,0 +1,13 @@
+﻿CREATE TABLE [dbo].[MarketRole]
```

Comment:
```suggestion
﻿CREATE TABLE [dbo].[EnergySupplier]
```
(or should the property `EnergySupplierId` be `MarketParticipantId`?)

Reply:
I think the naming of the table is as it should be. It hints that it holds data from the MarketRole domain.

Reply:
Let's talk.